index.html

```diff
@@ -11,11 +11,10 @@
   <body>
     <!-- Put your HTML here -->
     <div id="user-interface" class="centered">
-      
+      <div class="score">Correct Answers</div>
```

Comment:
create this element on init, no need to add html here. let's keep user-interface div empty